cli: TestConcurrentZip failed

cli.TestConcurrentZip [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3367054&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3367054&tab=artifacts#/) on master @ [2b8d42d76aba893dbef2e08b069a5ecf3f6bd566](https://github.com/cockroachdb/cockroach/commits/2b8d42d76aba893dbef2e08b069a5ecf3f6bd566):


```
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/1/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/1/crdb_internal.node_transaction_statistics.txt...
         [node 1] retrieving SQL data for crdb_internal.node_transactions...
         [node 1] retrieving SQL data for crdb_internal.node_transactions: done
         [node 1] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/1/crdb_internal.node_transactions.txt...
         [node 1] retrieving SQL data for crdb_internal.node_txn_stats...
        @@ -487,14 +491,18 @@
         [node 2] retrieving SQL data for crdb_internal.node_runtime_info: writing output: debug/nodes/2/crdb_internal.node_runtime_info.txt...
         [node 2] retrieving SQL data for crdb_internal.node_sessions...
         [node 2] retrieving SQL data for crdb_internal.node_sessions: done
         [node 2] retrieving SQL data for crdb_internal.node_sessions: writing output: debug/nodes/2/crdb_internal.node_sessions.txt...
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics...
        +[node 2] retrieving SQL data for crdb_internal.node_statement_statistics: creating error output: debug/nodes/2/crdb_internal.node_statement_statistics.txt.err.txt...
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics: done
        +[node 2] retrieving SQL data for crdb_internal.node_statement_statistics: last request failed: pq: internal error: read-stmt-stats: descriptor not found
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics: writing output: debug/nodes/2/crdb_internal.node_statement_statistics.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/2/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/2/crdb_internal.node_transaction_statistics.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transactions...
         [node 2] retrieving SQL data for crdb_internal.node_transactions: done
         [node 2] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/2/crdb_internal.node_transactions.txt...
         [node 2] retrieving SQL data for crdb_internal.node_txn_stats...
        @@ -766,14 +774,18 @@
         [node 3] retrieving SQL data for crdb_internal.node_runtime_info: writing output: debug/nodes/3/crdb_internal.node_runtime_info.txt...
         [node 3] retrieving SQL data for crdb_internal.node_sessions...
         [node 3] retrieving SQL data for crdb_internal.node_sessions: done
         [node 3] retrieving SQL data for crdb_internal.node_sessions: writing output: debug/nodes/3/crdb_internal.node_sessions.txt...
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics...
        +[node 3] retrieving SQL data for crdb_internal.node_statement_statistics: creating error output: debug/nodes/3/crdb_internal.node_statement_statistics.txt.err.txt...
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics: done
        +[node 3] retrieving SQL data for crdb_internal.node_statement_statistics: last request failed: pq: internal error: read-stmt-stats: descriptor not found
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics: writing output: debug/nodes/3/crdb_internal.node_statement_statistics.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/3/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/3/crdb_internal.node_transaction_statistics.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transactions...
         [node 3] retrieving SQL data for crdb_internal.node_transactions: done
         [node 3] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/3/crdb_internal.node_transactions.txt...
         [node 3] retrieving SQL data for crdb_internal.node_txn_stats...
    panic.go:613: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestConcurrentZip094912832
--- FAIL: TestConcurrentZip (2.99s)
```
<details><summary>Reproduce</summary>
<p>
To reproduce, try:

```bash
make stressrace TESTS=TestConcurrentZip PKG=./pkg/cli TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
/cc @cockroachdb/server knz
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestConcurrentZip.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

cli.TestConcurrentZip [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3373781&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3373781&tab=artifacts#/) on master @ [45ffbd022fb18124c610a2926dd7028a707f3f45](https://github.com/cockroachdb/cockroach/commits/45ffbd022fb18124c610a2926dd7028a707f3f45):


```
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/1/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/1/crdb_internal.node_transaction_statistics.txt...
         [node 1] retrieving SQL data for crdb_internal.node_transactions...
         [node 1] retrieving SQL data for crdb_internal.node_transactions: done
         [node 1] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/1/crdb_internal.node_transactions.txt...
         [node 1] retrieving SQL data for crdb_internal.node_txn_stats...
        @@ -487,14 +491,18 @@
         [node 2] retrieving SQL data for crdb_internal.node_runtime_info: writing output: debug/nodes/2/crdb_internal.node_runtime_info.txt...
         [node 2] retrieving SQL data for crdb_internal.node_sessions...
         [node 2] retrieving SQL data for crdb_internal.node_sessions: done
         [node 2] retrieving SQL data for crdb_internal.node_sessions: writing output: debug/nodes/2/crdb_internal.node_sessions.txt...
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics...
        +[node 2] retrieving SQL data for crdb_internal.node_statement_statistics: creating error output: debug/nodes/2/crdb_internal.node_statement_statistics.txt.err.txt...
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics: done
        +[node 2] retrieving SQL data for crdb_internal.node_statement_statistics: last request failed: pq: internal error: read-stmt-stats: descriptor not found
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics: writing output: debug/nodes/2/crdb_internal.node_statement_statistics.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/2/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/2/crdb_internal.node_transaction_statistics.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transactions...
         [node 2] retrieving SQL data for crdb_internal.node_transactions: done
         [node 2] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/2/crdb_internal.node_transactions.txt...
         [node 2] retrieving SQL data for crdb_internal.node_txn_stats...
        @@ -766,14 +774,18 @@
         [node 3] retrieving SQL data for crdb_internal.node_runtime_info: writing output: debug/nodes/3/crdb_internal.node_runtime_info.txt...
         [node 3] retrieving SQL data for crdb_internal.node_sessions...
         [node 3] retrieving SQL data for crdb_internal.node_sessions: done
         [node 3] retrieving SQL data for crdb_internal.node_sessions: writing output: debug/nodes/3/crdb_internal.node_sessions.txt...
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics...
        +[node 3] retrieving SQL data for crdb_internal.node_statement_statistics: creating error output: debug/nodes/3/crdb_internal.node_statement_statistics.txt.err.txt...
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics: done
        +[node 3] retrieving SQL data for crdb_internal.node_statement_statistics: last request failed: pq: internal error: read-stmt-stats: descriptor not found
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics: writing output: debug/nodes/3/crdb_internal.node_statement_statistics.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/3/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/3/crdb_internal.node_transaction_statistics.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transactions...
         [node 3] retrieving SQL data for crdb_internal.node_transactions: done
         [node 3] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/3/crdb_internal.node_transactions.txt...
         [node 3] retrieving SQL data for crdb_internal.node_txn_stats...
    panic.go:613: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestConcurrentZip270721241
--- FAIL: TestConcurrentZip (2.90s)
```
<details><summary>Reproduce</summary>
<p>
To reproduce, try:

```bash
make stressrace TESTS=TestConcurrentZip PKG=./pkg/cli TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
/cc @cockroachdb/server knz
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestConcurrentZip.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

cli.TestConcurrentZip [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3377491&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3377491&tab=artifacts#/) on master @ [6700cf65e82a029a1ff76a75250c709aa9637170](https://github.com/cockroachdb/cockroach/commits/6700cf65e82a029a1ff76a75250c709aa9637170):


```
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/1/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 1] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/1/crdb_internal.node_transaction_statistics.txt...
         [node 1] retrieving SQL data for crdb_internal.node_transactions...
         [node 1] retrieving SQL data for crdb_internal.node_transactions: done
         [node 1] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/1/crdb_internal.node_transactions.txt...
         [node 1] retrieving SQL data for crdb_internal.node_txn_stats...
        @@ -487,14 +491,18 @@
         [node 2] retrieving SQL data for crdb_internal.node_runtime_info: writing output: debug/nodes/2/crdb_internal.node_runtime_info.txt...
         [node 2] retrieving SQL data for crdb_internal.node_sessions...
         [node 2] retrieving SQL data for crdb_internal.node_sessions: done
         [node 2] retrieving SQL data for crdb_internal.node_sessions: writing output: debug/nodes/2/crdb_internal.node_sessions.txt...
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics...
        +[node 2] retrieving SQL data for crdb_internal.node_statement_statistics: creating error output: debug/nodes/2/crdb_internal.node_statement_statistics.txt.err.txt...
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics: done
        +[node 2] retrieving SQL data for crdb_internal.node_statement_statistics: last request failed: pq: internal error: read-stmt-stats: descriptor not found
         [node 2] retrieving SQL data for crdb_internal.node_statement_statistics: writing output: debug/nodes/2/crdb_internal.node_statement_statistics.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/2/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 2] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/2/crdb_internal.node_transaction_statistics.txt...
         [node 2] retrieving SQL data for crdb_internal.node_transactions...
         [node 2] retrieving SQL data for crdb_internal.node_transactions: done
         [node 2] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/2/crdb_internal.node_transactions.txt...
         [node 2] retrieving SQL data for crdb_internal.node_txn_stats...
        @@ -766,14 +774,18 @@
         [node 3] retrieving SQL data for crdb_internal.node_runtime_info: writing output: debug/nodes/3/crdb_internal.node_runtime_info.txt...
         [node 3] retrieving SQL data for crdb_internal.node_sessions...
         [node 3] retrieving SQL data for crdb_internal.node_sessions: done
         [node 3] retrieving SQL data for crdb_internal.node_sessions: writing output: debug/nodes/3/crdb_internal.node_sessions.txt...
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics...
        +[node 3] retrieving SQL data for crdb_internal.node_statement_statistics: creating error output: debug/nodes/3/crdb_internal.node_statement_statistics.txt.err.txt...
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics: done
        +[node 3] retrieving SQL data for crdb_internal.node_statement_statistics: last request failed: pq: internal error: read-stmt-stats: descriptor not found
         [node 3] retrieving SQL data for crdb_internal.node_statement_statistics: writing output: debug/nodes/3/crdb_internal.node_statement_statistics.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics...
        +[node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: creating error output: debug/nodes/3/crdb_internal.node_transaction_statistics.txt.err.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: done
        +[node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: last request failed: pq: internal error: read-txn-stats: descriptor not found
         [node 3] retrieving SQL data for crdb_internal.node_transaction_statistics: writing output: debug/nodes/3/crdb_internal.node_transaction_statistics.txt...
         [node 3] retrieving SQL data for crdb_internal.node_transactions...
         [node 3] retrieving SQL data for crdb_internal.node_transactions: done
         [node 3] retrieving SQL data for crdb_internal.node_transactions: writing output: debug/nodes/3/crdb_internal.node_transactions.txt...
         [node 3] retrieving SQL data for crdb_internal.node_txn_stats...
    panic.go:613: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestConcurrentZip242077361
--- FAIL: TestConcurrentZip (2.91s)
```
<details><summary>Reproduce</summary>
<p>
To reproduce, try:

```bash
make stressrace TESTS=TestConcurrentZip PKG=./pkg/cli TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
/cc @cockroachdb/server knz
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestConcurrentZip.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cli/zip_test.go
+++ b/pkg/cli/zip_test.go
@@ -164,10 +164,10 @@ func TestConcurrentZip(t *testing.T) {
 	ctx := context.Background()
 
 	// Three nodes. We want to see what `zip` thinks when one of the nodes is down.
+	params, _ := tests.CreateTestServerParams()
+	params.Insecure = true
 	tc := testcluster.StartTestCluster(t, 3,
-		base.TestClusterArgs{ServerArgs: base.TestServerArgs{
-			Insecure: true,
-		}})
+		base.TestClusterArgs{ServerArgs: params})
 	defer tc.Stopper().Stop(ctx)
 
 	// Zip it. We fake a CLI test context for this.
@@ -270,11 +270,14 @@ func TestUnavailableZip(t *testing.T) {
 	}
 
 	// Make a 2-node cluster, with an option to make the first node unavailable.
+	params, _ := tests.CreateTestServerParams()
+	params.Insecure = true
 	tc := testcluster.StartTestCluster(t, 2, base.TestClusterArgs{
 		ServerArgsPerNode: map[int]base.TestServerArgs{
 			0: {Insecure: true, Knobs: base.TestingKnobs{Store: knobs}},
 			1: {Insecure: true},
 		},
+		ServerArgs: params,
 	})
 	defer tc.Stopper().Stop(context.Background())
 
@@ -452,7 +455,9 @@ func TestZipRetries(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	s, _, _ := serverutils.StartServer(t, base.TestServerArgs{Insecure: true})
+	params, _ := tests.CreateTestServerParams()
+	params.Insecure = true
+	s, _, _ := serverutils.StartServer(t, params)
 	defer s.Stopper().Stop(context.Background())
 
 	dir, cleanupFn := testutils.TempDir(t)

